payment-manager/src/main/resources/application.yaml

```diff
@@ -0,0 +1,14 @@
+server:
+  port: 8082
+spring:
+  application:
+    name: users
+  flyway:
+    schemas: users
```

Comment:
It's so annoying when you copy/paste code and expect us to review your code instead of doing it yourself before creating a PR.